src/olympia/activity/serializers.py

```diff
@@ -24,7 +24,8 @@ def __init__(self, *args, **kwargs):
         self.to_highlight = kwargs.get('context', []).get('to_highlight', [])
 
     def get_comments(self, obj):
-        return getattr(obj.log(), 'sanitize', obj.details['comments'])
+        return getattr(obj.log(), 'sanitize',
+                       obj.details['comments'] if obj.details else obj.log())
 
```

Comment:
honestly, this is two decision-branches in one line an deserves to be rewritten as some easier to understand code with a proper if statement and a default branch imho.